ftdetect/json.lua

```diff
@@ -0,0 +1,3 @@
+vim.cmd [[
+ au BufRead,BufNewFile tsconfig.json set filetype=jsonc
```

Comment:
Yes, indeed, it is an exception for json.
When you do `yarn tsc --init` inside a typescript project you get a `tsconfig.json` rather than `tsconfig.jsonc`. 
I think many other editors treat it as a jsonc file but (neo)vim doesnt.

Reply:
> Shouldn't the jsonc files have a different extension from json files ?

They sadly don't.